warehouse/models/staging/transit_database/stg_transit_database__gtfs_datasets.sql

```diff
@@ -87,6 +87,13 @@ stg_transit_database__gtfs_datasets AS (
         authorization_header_parameter_name,
         header_secret_key_name,
         url_to_encode,
+        manual_check__link_to_dataset_on_website,
+        manual_check__accurate_shapes,
+        manual_check__data_license,
+        manual_check__authentication_acceptable,
+        manual_check__stable_url,
+        manual_check__localized_stop_tts,
```

Comment:
What his (stop TTS) already in the external table? Don't see it added here

Reply:
I think it's here https://github.com/cal-itp/data-infra/pull/2197/files#diff-967c2a19f93fc05bf18c018f3cebf57245be1b197007fbf96e57653c978df589R166

Reply:
Oh sorry my bad, I missed it because it wasn't in the list he requested. Good catch to you